utils/metrics.py

```diff
@@ -71,10 +73,23 @@ def report_classification(pred, label, batch_size, metrics_dict, ignore_index=-1
     return metrics_dict
 
 
-def iou(pred, target, batch_size, metrics_dict):
-    """Calculate the intersection over union (or Jaccard index) between two datasets.
-    The Jaccard distance (or dissimilarity) would be 1-iou.
-    """
-    iou = jaccard_similarity_score(target.cpu(), pred.cpu(), normalize=True)
-    metrics_dict['iou'].update(iou, batch_size)
-    return metrics_dict
+def iou(pred, label, batch_size, num_classes, metric_dict, only_present=True):
```

Comment:
Awesome! well done!  
Have you noticed some overhead caused by the time it takes to compute the IoU?

Reply:
Very negligible at the test stage, if it is computed all to frequently at the validation set then we may notice some overhead. At the moment it is easy to live with.